tox.ini

```diff
@@ -54,7 +54,15 @@ commands_pre =
 
 [testenv:type]
 description = run type check on code base
-extras = typing
+skip_install = true
+setenv =
+    PYTHONWARNDEFAULTENCODING =
+deps =
+    importlib-metadata>=5.1
+    mypy==1.2.0
+    tomli
+    typing-extensions>=3.7.4.3
```

Comment:
Not needed unless it's a runtime dependency. Mypy pulls typing-extensions in automatically.